component/vshn_postgres.jsonnet

```diff
@@ -37,6 +37,8 @@ local connectionSecretKeys = [
   'POSTGRESQL_PORT',
   'POSTGRESQL_USER',
   'POSTGRESQL_PASSWORD',
+  'PRIMARY_IP',
+  'REPLICAS_IP',
```

Comment:
I wouldn't expose the replica IPs, we also hide them internally.

Reply:
I think that replicas could be very important for our customers, You normally don't want to do analytics using master node/instance, this is why I decided to expose REPLICA_IP, also it might be important for customers to make fast snapshot of database for local development purposes - also replica is much better choice.

What are counter arguments?